cns/restserver/restserver.go

```diff
@@ -135,6 +137,21 @@ type containerstatus struct {
 	VfpUpdateComplete             bool // True when VFP programming is completed for the NC
 }
 
+type multiContainerStatus map[string]containerstatus
+
+func (mcs *multiContainerStatus) GetUnsuccessfulStatusErrors() string {
+	var unsuccessfulStatuses []string
+	for ncID := range *mcs {
+		ncStatus := (*mcs)[ncID].CreateNetworkContainerRequest.NCStatus
+		if ncStatus != cns.NCUpdateSuccess {
+			unsuccessfulStatus := fmt.Sprintf("Expected status for NC %s to be %s but got %s", ncID, string(cns.NCUpdateSuccess), string(ncStatus))
+			unsuccessfulStatuses = append(unsuccessfulStatuses, unsuccessfulStatus)
+		}
+	}
+
+	return strings.Join(unsuccessfulStatuses, "\n")
```

Comment:
Also, it's unusual to join by newlines for errors since the log lines themselves are newline separated